README.md

```diff
@@ -466,6 +466,8 @@ These directives are the `awPreviousStep`, `asNextStep` and `awGoToStep` directi
 The `awGoToStep` directive needs to receive an input, which tells the wizard, to which step it should navigate,
 when the element with the `awGoToStep` directive has been clicked.
 
+> Please be aware that `awGoToStep` will not proceed to another step unless the intermediate steps are marked as optional, using the directive `awOptionalStep`, or has been completed, either by the user or by marking them as completed by default with the directive `awCompletedStep`.
```

Comment:
Two suggestions:
- I think we should split this sentence into two to make it easier to understand. It contains multiple subclauses which make it hard to follow the exceptions to the rule
- I'm not sure whether we should use a citation `> ...` here. In other places in the README we haven't used citations so I think it is better to continue not using them